lib/client.ts

```diff
@@ -42,52 +43,58 @@ export default class Client {
     return this.post(URL.multicast, {
       messages: toArray(messages),
       to,
-    });
+    }).then(this.checkJSON);
```

Comment:
We needn't care about response here (anyway the response will be empty, thus return type is `any`), so just remove the check.